### DISCUSSION TITLE
The autogen-agentchat package is not up to date.

### DISCUSSION THREAD

**Discussion by autogen-user** [microsoft/autogen#3805] [General]
When I check the following file: (https://github.com/microsoft/autogen/blob/0.2/autogen/agentchat/groupchat.py), I can see in the "GroupChat class" that there is a parameter/field named: "select_speaker_auto_model_client_cls".
This field/parameter is also mentioned in the AutoGen documentation here: https://microsoft.github.io/autogen/0.2/docs/reference/agentchat/groupchat

However, when I install the Python package: "autogen-agentchat~=0.2." from PyPI (https://pypi.org/project/autogen-agentchat/),
the parameter/field is not present in the groupchat.py file! Why is there this discrepancy? Why is this parameter mentioned in the documentation and in some threads, but it is not included when installing "autogen-agentchat"?

**Comment by ekzhu** [comment]
Dear @autogen-user, this happens when the release lags behind the main branch. We will make a release soon.
